### ISSUE TITLE
FirstnameLastname reference doesn't resolve

### ISSUE THREAD

**Issue by tomten** [MicrosoftDocs/windows-itpro-docs#6902]
The documentation says

> net localgroup "Remote Desktop Users" /add "AzureAD\the-UPN-attribute-of-your-user", where _FirstnameLastname_ is the name of the user profile in C:\Users, 

However, there is no FirstnameLastname in the referenced material. I'm guessing it should say 

> where _the-UPN-attribute-of-your-user_ is the name of the user profile in C:\Users, 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 72e1cc18-3fc2-d7fa-7ff3-20fc9a3596d4
* Version Independent ID: 8ed24664-ce8c-7c08-654a-d3ef267f274f
* Content: [Connect to remote Azure Active Directory-joined PC (Windows 10) - Windows Client Management](https://docs.microsoft.com/en-us/windows/client-management/connect-to-remote-aadj-pc)
* Content Source: [windows/client-management/connect-to-remote-aadj-pc.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/client-management/connect-to-remote-aadj-pc.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @Dansimp
* Microsoft Alias: **dansimp**

**Comment by illfated** [comment]
Good point. Some of the other pages containing examples of how that command should look, use the following phrasing:
```
net localgroup “Remote Desktop Users” /add “AzureAD\john.doe@domain.com”
```
This shows what the expected name format could be, but the question remains whether the username has to be in the format `user.name@domain.ext` or if it can be only "AzureAD\username" which is a name format backwards compatible with older Windows domain names and local user names.

I am thinking about opening a PR to have that line changed as you suggest, although I assume I might get suggestions for a different username format in the PR.

**Comment by illfated** [comment]
Worth noticing: 
Looking at the page [How to manage the local administrators group on Azure AD joined devices](https://docs.microsoft.com/en-us/azure/active-directory/devices/assign-local-admin) (direct link: https://docs.microsoft.com/en-us/azure/active-directory/devices/assign-local-admin#manually-elevate-a-user-on-a-device), I see the following phrasing used:

## Manually elevate a user on a device 

In addition to using the Azure AD join process, you can also manually elevate a regular user to become a local administrator on one specific device. This step requires you to already be a member of the local administrators group. 

Starting with the **Windows 10 1709** release, you can perform this task from **Settings -> Accounts -> Other users**. Select **Add a work or school user**, enter the user's UPN under **User account** and select *Administrator* under **Account type**  
 
Additionally, you can also add users using the command prompt:

- If your tenant users are synchronized from on-premises Active Directory, use `net localgroup administrators /add "Contoso\username"`.
- If your tenant users are created in Azure AD, use `net localgroup administrators /add "AzureAD\UserUpn"`

**Comment by illfated** [comment]
Instead of "UserUpn" I am tempted to suggest "UserPrincipalName" instead, although I am not certain that every admin user is perfectly familiar with the definition of "User Principal Name" or UPN. Maybe it is just as simple to change the casing to "userUPN" to highlight that the name is the initialism for "User Principal Name".

**Comment by illfated** [comment]
(See also PR #6906 for more details)
